packages/shared/src/components/feeds/MobileFeedActions.tsx

```diff
@@ -42,7 +42,6 @@ export function MobileFeedActions(): ReactElement {
           onOpenFeedFilters={() =>
             openModal({
               type: LazyModal.FeedFilters,
-              persistOnRouteChange: true,
```

Comment:
Best to check with @sshanzel I think.
Is it not something on submit you get redirected, but we wanted to persist the modal open?

Reply:
There's no "submit" on the modal. And I tested in preview that can toggle any setting in the modal without it closing while I have `persistOnRouteChange` disabled.

Reply:
I had to disable it so that modal would close when I navigate to the profile page 🙈
Could remove the `Link` and have it behave like normal link, that does a full navigation, but it's slower.

Reply:
We can maybe manually close the modal when Link is clicked using `onClick`?

Reply:
I got a circular import error when I tried to import useLazyModal to get access to closeModal 🙈

Reply:
actually, think I fixed it, just changed import to type in useLazyModal